spec/classes/stubbed_command_spec.rb

```diff
@@ -1,5 +1,4 @@
-require 'English'
```

Comment:
This and all below are simply inclusion of a common spec helper file, for coverage.